systems/inventory/component/pkg/db/model.go

```diff
@@ -51,7 +51,7 @@ func (c ComponentCategory) Value() (uint8, error) {
 }
 
 func (c ComponentCategory) String() string {
```

Comment:
We need to move ComponentCategory type to `systems/common/pb/ukama`. We recently move most of the custom types there. I think this one left.